api/v1alpha1/autoscaler_types.go

```diff
@@ -100,7 +69,7 @@ type Trigger struct {
 	Type TriggerType `json:"type"`
 
 	// Condition set the condition when to trigger scaling
-	Condition TriggerCondition `json:"condition"`
+	Condition map[string]string `json:"condition"`
```

Comment:
In the first discussion, we intend to explicitly set the condition fields. In the new modification, we can only reply on Autoscaler docs to teach users how to use the trait.

Reply:
After I digging into the implementation, we have no ideas but set it as `map[string]string`, if not so, we have no extentions.